provider/pkg/gen/dotnet-templates/helm/ChartBase.cs

```diff
@@ -335,7 +338,7 @@ private static string EscapeArguments(string argument)
 
                     case '\"':
                         // Escape any preceding backslashes
-                        escapedArgument.Append(new string('\\', backslashCount));
+                        escapedArgument.Append('\\', backslashCount);
```

Comment:
`StringBuilder` supports appending a character a number of times, so no need to allocate a string to do that.